opengemini_client/client_impl.py

```diff
@@ -128,8 +131,25 @@ def ping(self, idx: int):
             return None
         return Error(f"ping openGeminiDB status is {resp.status_code}")
 
-    def query(self, query: Query) -> tuple:
-        return ()
+    def query(self, query: Query):
+
+        if query.command.lower().startswith("select") and "into" not in query.command.lower():
```

Comment:
This logic is incorrect. For example, drop database will be judged as a GET type

Reply:
ok